src/frontend/magnify/src/components/app/MagnifyProvider/index.tsx

```diff
@@ -41,26 +27,17 @@ const queryClient = new QueryClient({
 });
 
 export interface MagnifyProviderProps {
-  children?: React.ReactNode;
   initialUser?: User;
   translations?: any;
+  locale?: string;
 }
 
-const locale = 'en-US';
-
-export function MagnifyProvider({ ...props }: MagnifyProviderProps) {
-  const [translations, setTranslations] = useState<
-    Maybe<Record<string, string> | Record<string, MessageFormatElement[]>>
-  >({});
-
-  useEffect(() => {
-    getTranslation(locale).then((initTranslation) => {
-      setTranslations(initTranslation);
-    });
-  }, []);
-
+export function MagnifyProvider({
+  locale = 'en-US',
```

Comment:
Hmm.. `locale` is optional within `TranslationProvider` and have already a default value (which is French by the way) so I think locale should be optional here.